### DISCUSSION TITLE
Adding user-defined variability on fraction vascular

### DISCUSSION THREAD

**Discussion by prvmalik** [Open-Systems-Pharmacology/Forum#1656] [Questions & Problems]
Hi 

I'd like to add user-defined variability to fraction vascular. I've gone to Mobi and checked "can be varied in a population" for the fraction vascular in each organ and then transferred the simulation back to PK-Sim for a population simulation. 

But now I can't remember what to do next so that the fraction vascular parameters actually show up to enable user-defined variability. Thanks in advance for the help.

**Comment by Yuri05** [comment]
The problem is that `Fraction vascular` is initially marked as **read-only** in PK-Sim.
When exporting to MoBi, this information is kept internally (but not accessible in the UI).
When sending back to PK-Sim, parameter becomes read-only again and cannot be distributed in population simulation.

As a workaround: delete the parameter `Fraction vascular` in MoBi, then recreate it and check _Can be varied in population_.
Now when sending the simulation to PK-Sim, parameter can be distributed in a pop simulation
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/cb2c542d-9ac8-4b16-89bd-40467570744a)

**Reply by prvmalik** [reply]
Thanks. I thought also to create an organism- level scaling factor parameter for fv and vary that. Appreciate it!

**Reply by nguyen-tranbach** [reply]
How do you delete then recreate the parameter? I struggle while following the instructions in Mobi documents
